Index.html

```diff
@@ -32,8 +32,53 @@ <h2>I’m a software developer</h2>
          <li><a><img src="media/github.svg" alt="github link"></a></li>
          <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
         </ul>
+
+        <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
  
       </div>
+      <section class="work-container">
+        <div class="my-work-text">
+          <h2>My Recent Works</h2>
+          <hr>
+       </div>
```

Comment:
- [ ] Nice work implementing the title :+1: However there is too much space between the title and the works (see image), please fixed this in order to follow the design.

This is what your website looks like:
![Screenshot from 2023-03-25 01-29-50](https://user-images.githubusercontent.com/91142432/227679623-8b199edd-26c8-4052-91a0-d134c7e3d7c3.png)
And this is what it suppose to look like:
![Screenshot from 2023-03-25 01-32-52](https://user-images.githubusercontent.com/91142432/227679654-4acc3e0b-97d2-4ffa-803e-da685949fd21.png)

Reply:
resolved